TinyLoRa.h

```diff
@@ -37,7 +37,12 @@
 #define TINY_LORA_H
 
 #include <Arduino.h>
-#include <avr/pgmspace.h>
+#ifdef ARDUINO_ARCH_AVR
```

Comment:
Instead of two `#ifdef`s, could you use an `if/elif` conditional statement, like:

```
#if defined(ARDUINO_ARCH_AVR)
  #include <avr/pgmspace.h>
#elif defined(ARDUINO_ARCH_ESP32)
  #include <pgmspace.h>
#endif
```